app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def index
+
+      channels = Channel.all.map { |channel| channel.id }
```

Comment:
If I understand this correctly, this selects all the columns and then does a parse. `Channel.pluck(:id)` should tell the database to only get the `id` column.

Reply:
Added ✔️